src/site/layouts/faq_multiple_q_a.drupal.liquid

```diff
@@ -52,17 +52,16 @@
             <div class="usa-accordion-bordered">
               <ul class="usa-unstyled-list">
                 {% for fieldQA in fieldQAGroup.entity.fieldQAs %}
-                  <li>
+                  <li data-faq-entity-id="{{ fieldQA.entity.entityId }}">
                     <button type="button" aria-controls="{{ fieldQA.entity.entityBundle }}-{{ fieldQA.entity.entityId }}"
                       class="usa-accordion-button usa-button-unstyled"
                       aria-expanded="false">{{ fieldQA.entity.title }}</button>
                     <div id="{{ fieldQA.entity.entityBundle }}-{{ fieldQA.entity.entityId }}" class="usa-accordion-content"
                       aria-hidden="true">
-                      {% for block in fieldQA.entity.fieldAnswer %}
-                        {% assign bundleComponent = "src/site/paragraphs/" | append: block.entity.entityBundle %}
-                        {% assign bundleComponentWithExtension = bundleComponent | append: ".drupal.liquid" %}
-                        {% include {{ bundleComponentWithExtension }} with entity = block.entity %}
-                      {% endfor %}
+                      {% assign fieldAnswer = fieldQA.entity.fieldAnswer %}
+                      {% assign bundleComponent = "src/site/paragraphs/" | append: fieldAnswer.entity.entityBundle %}
+                      {% assign bundleComponentWithExtension = bundleComponent | append: ".drupal.liquid" %}
+                      {% include {{ bundleComponentWithExtension }} with entity = fieldAnswer.entity %}
```

Comment:
`fieldQA.entity.fieldAnswer` is no longer an array either. It's just an object.

![image](https://user-images.githubusercontent.com/1915775/96593992-217c2a80-12b8-11eb-9537-afb988bc42ca.png)